app/Resources/views/login/login.html.twig

```diff
@@ -8,6 +8,10 @@
     <link href="{{ asset('css/loginPage.css') }}" rel="stylesheet" />
 {% endblock %}
 
+{% block navbar %}
```

Comment:
Warum muss dieser Block hier rein?

Reply:
Damit der Inhalt der base.html.twig mit diesem leeren String überschrieben wird. Sonst wäre die Navbar für JEDEN einsehbar. Wollen wir das? So konzentriert sich die Loginseite wirklich auf den Login und die Funktionen werden nach dem Login sichtbar.
Künftig würde ich mit ACLs auch nur die Obermenüs in der Navbar anzeigen, zu denen man Zugriffsrechte besitzt, um die Error 403 Forbidden Seite zu vermeiden.

Reply:
You are right!